src/pages/SigningOfficialResearchers.js

```diff
@@ -20,13 +19,7 @@ export default function SigningOfficialResearchers() {
       try {
         setIsLoading(true);
         const soUser = await User.getMe();
-        const soPromises = await Promise.all([
-          User.list(USER_ROLES.signingOfficial),
-          User.getUnassignedUsers()
```

Comment:
This was the only usage of `User.getUnassignedUsers()` so we can remove it, along with deprecating the API method in consent: `GET /api/user/institution/unassigned`

Reply:
it's also used in `SigningOfficialDataSubmitters.js`; is that deprecated code?

Reply:
Looks like I was incorrect ... it is used in three places in develop right now, not sure how I missed them.
* SigningOfficialConsole
* SigningOfficialDataSubmitters
* SigningOfficialResearchers